addon/mixin.js

```diff
@@ -114,5 +114,9 @@ export default Ember.Mixin.create({
     if (empty(this.buffer)) {
       this.set('hasBufferedChanges', false);
     }
+  },
+
+  _compareValues(a, b) {
```

Comment:
Since you're intending the consumer to override this in some situations, why did you go with the `_` naming convention here?